core/src/components/picker-column-option/picker-column-option.tsx

```diff
@@ -75,8 +124,8 @@ export class PickerColumnOption implements ComponentInterface {
           ['option-disabled']: disabled,
         })}
       >
-        <button tabindex="-1" aria-label={ariaLabel} disabled={disabled}>
-          <slot>{value}</slot>
```

Comment:
This causes issues with Playwright's `toHaveText` since both the slotted content and the default content are accounted for. Given that we don't have this behavior today I've opted to remove it.